### PR TITLE
fix: apply class to autocomplete popup for no suggestions state

### PR DESCRIPTION
**Pull request by akoreman**
*Issue #, if available:* NA

*Description of changes:* Apply class when autocomplete popup is in the 'no suggestions' state to allow separate styling of that state.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

Pull Request Checklist:
* [x] No backwards incompatible changes were made to Ace's public interface which is defined through the main typings file (`ace.d.ts`) and its references:
    * https://github.com/ajaxorg/ace/blob/master/ace.d.ts 
    * https://github.com/ajaxorg/ace/blob/master/ace-modes.d.ts 
    * https://github.com/ajaxorg/ace/blob/master/ace-extensions.d.ts

### CODE REVIEW

File: src/autocomplete_test.js [hunk 1]
```diff
@@ -462,12 +462,43 @@ module.exports = {
...
+        ];
+
+        user.type("app");
+
+        // Open autocompletion via key-binding and verify no empty message class
+        user.type("Ctrl-Space");
+        assert.equal(editor.completer.popup.isOpen, true);
+        assert.equal(editor.completer.popup.data[0].caption, "append");
+        assert.ok(!editor.completer.popup.renderer.container.classList.contains("ace_empty-message"));
+
```
Comment (whazor) [inline src/autocomplete_test.js:499]: It would be nice to also test the reverse `empty -> results`

Comment (akoreman) [reply src/autocomplete_test.js:499]: so trigger it 1 time without results (check class there) and then trigger with results and check class not there?

Comment (akoreman) [reply src/autocomplete_test.js:508]: Improved the test to test that scenario as well

File: src/autocomplete_test.js [hunk 2]
```diff
@@ -462,12 +462,52 @@ module.exports = {
...
+                            caption: "append",
+                            value: "append"
+                        }
+                    ];
+                    callback(null, completions);
+                }
+            }
+        ];
+
+        user.type("b");
```
[the remark this answers is not in the document]
Comment (oykuyilmaz) [reply src/autocomplete_test.js:492]: I retract my comment, I've missed that you register the completer explicitly.

**Review by oykuyilmaz — APPROVED** [review]
